### ISSUE TITLE
Locast2plex Service not starting.

### ISSUE THREAD

**Issue by JimSpeedo** [tgorgdotcom/locast2plex#129]
Installed as a windows service.

C:\locast2plex\main.py -u:usernamexxx -p:passwordxxx --debug --addy:127.0.0.1 --port:6077

Yesterday everything was working with no issues. Today I cant get the service started.  I even removed and re added the service and still the same issue. Just says Service could not start but did not generate an error. Nothing really in the logs that helped me either. Starnge that it just stopped working as it has been working incredibly well for months now!

**Comment by wdreiling** [comment]
Having the same trouble.  Can't seem to manually run either.  Haven't made any changes for a month or two.

**Comment by ratherDashing** [comment]
same here

**Comment by craigarno** [comment]
Ditto

**Comment by Neotribal** [comment]
Same. Running main.py I get an error "Error in function login: Not Found"

**Comment by scumola** [comment]
Ditto.  Locast must have changed something.  Running the 0.5.3 docker container.

**Comment by scumola** [comment]
Starting around 6am Colorado-time, my recordings are just a video banner saying that I've exceeded my number of active streams for my account.  I checked and I was only recording the one stream, so something is wrong at Locast or they've blacklisted me maybe.

**Comment by mbtfeedback** [comment]
Same issue here. And when I decided to remove the container and start from scratch, no love. It just starts and exits.

**Comment by c0deweaver** [comment]
This issue should be merged into #128 

TL;DR: 
@tgorgdotcom has already opened a support ticket with locast
Tested adding a user agent to API request to login to no avail. 
Current state is waiting on locast.

**Comment by c0deweaver** [comment]
This should be resolved by [this commit](https://github.com/tgorgdotcom/locast2plex/commit/8cff4fad49cb7b5899b92512f4218cf093466819).

Pull the latest and test it. Make sure you can log into the web interface and are not rate limited. If you are rate limited you will need to wait an hour before trying to log in again.

**Comment by tgorgdotcom** [comment]
Duplicate of #128
